examples/tutorials/text.py

```diff
@@ -0,0 +1,139 @@
+# The size, family/weight, and colour of an annotation can be specified using
+# the ``font`` argument.
+#
+# A list of all recognised fonts can be found at
+# :gmt-docs:`cookbook/postscript_fonts.html`, including details of how to use
+# non-default fonts.
+
+fig = pygmt.Figure()
+with pygmt.config(MAP_FRAME_TYPE="plain"):
+    fig.basemap(region=[108, 120, -5, 8], projection="M20c", frame="a")
+fig.coast(land="black", water="skyblue")
+
+# Customising the font style
+fig.text(text="BORNEO", x=114.0, y=0.5, font="22p,Helvetica-Bold,white")
+
+fig.show()
+
+###############################################################################
+# Plotting from a text file
+# -------------------------
+#
+# It is also possible to add annotations from a file containing `x`, `y`, and
+# `text` fields. Here we give a complete example.
+
+fig = pygmt.Figure()
+with pygmt.config(MAP_FRAME_TYPE="plain"):
+    fig.basemap(region=[108, 120, -5, 8], projection="M20c", frame="a")
+fig.coast(land="black", water="skyblue")
+
+# Create space-delimited file
+with open("examples.txt", "w") as f:
+    f.write("114 0.5 0 22p,Helvetica-Bold,white CM BORNEO\n")
+    f.write("119 3.25 0 12p,Helvetica-Bold,black CM CELEBES SEA\n")
+    f.write("112 -4.6 0 12p,Helvetica-Bold,black CM JAVA SEA\n")
+    f.write("112 6 40 12p,Helvetica-Bold,black CM SOUTH CHINA SEA\n")
+    f.write("119.12 7.25 -40 12p,Helvetica-Bold,black CM SULU SEA\n")
+    f.write("118.4 -1 65 12p,Helvetica-Bold,black CM MAKASSAR STRAIT\n")
+
+# Plot region names / sea names from a text file
+fig.text(textfiles="examples.txt", angle=True, font=True, justify=True)
+
+# Cleanups
+os.remove("examples.txt")
+
+fig.show()
+
+###############################################################################
+# ``justify`` argument
+# --------------------
+#
+# ``justify`` is used to define the anchor point for the bounding box for text
+# being added to a plot. The following code segment demonstrates the
+# positioning of the anchor point relative to the text.
+#
+# The anchor is specified with a two letter (order independent) code, chosen
+# from:
+# * Vertical anchor: T(op), M(iddle), B(ottom)
+# * Horizontal anchor: L(eft), C(entre), R(ight)
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 3, 0, 3], projection="X10c", frame=["WSne", "af0.5g"])
+for position in ("TL", "TC", "TR", "ML", "MC", "MR", "BL", "BC", "BR"):
+    fig.text(
+        text=position,
+        position=position,
+        font="28p,Helvetica-Bold,black",
+        justify=position,
+    )
+fig.show()
+
+###############################################################################
+# ``angle`` argument
+# ------------------
+# ``angle`` is an optional argument used to specify the clockwise rotation of
+# the text from the horizontal.
+
+fig = pygmt.Figure()
+fig.basemap(region=[0, 4, 0, 4], projection="X10c", frame="WSen")
+for i in range(0, 360, 30):
+    fig.text(text=f"`          {i} Degrees", x=2, y=2, justify="LM", angle=i)
```

Comment:
`@.` is a shorthand for the degree symbol.
```suggestion
    fig.text(text=f"`          {i}@.", x=2, y=2, justify="LM", angle=i)
```

![image](https://user-images.githubusercontent.com/3974108/96062031-b0c1b200-0e62-11eb-81c3-5cb2338541ad.png)